app/models/pawn.rb

```diff
@@ -6,4 +6,39 @@ def image
       '&#9823;'
     end
   end
+
+  def valid_move?(x, y)
+    dx = delta_x(x_position, x)
+    dy = delta_y(y_position, y)
+
+    return false if blocked?(x, y)
+
+    # pawn can't move horizontally
+    return false if dx != 0 && dy.zero?
+
+    # pawn moves vertically 1 space
+    return true if dx.zero? && dy == 1
+    return false if color == 'white' && y_position < y
+    return false if color == 'black' && y_position > y
+    false
+  end
+
+  def capture_move?(x, y)
+    x_diff = (x_position - x).abs
+    y_diff = (y_position - y).abs
+    captured_piece = square_occupied?(x, y)
+    return true if x_diff == 1 && y_diff == 1
+    return false if captured_piece.blank?
+    # return false if captured_piece.color ==
+  end
+
+  def first_move?(y)
+    (color == 'white' && y_position == 6) || (color == 'black' && y_position == 1)
+    y_diff = (y_position - y).abs
+    return true if y_diff == 1 || y_diff == 2
```

Comment:
This line totally works, but Rubocop is pushing you towards an array inclusion check - that looks like this instead:

```ruby
[1, 2].includes?(y_diff)
```

So basically it's just checking to see if the value of `y_diff` is included in the array of `[1, 2]`.  For something small like this, either way is pretty readable, however if you had to check more values then it starts to be more readable in the array format:

```ruby
y_diff == 1 || y_diff == 2 || y_diff == 5 || y_diff == 8 || y_diff == 11 || y_diff == 14 || y_diff == 15 || y_diff == 16

# or

[1, 2, 5, 8, 11, 14, 15, 16].includes?(y_diff)
```